lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'dish'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :order_list, :total
+
+  def initialize
+    @order_list = []
+    @total = 0
+  end
+
+  def add(dish, quantity)
+    add_order_to_list(dish, quantity)
```

Comment:
`add_order_to_list` is only called within this method, so we can decrease the number of methods (and therefore the cognitive overload for someone who may read your code) by pulling the logic out of `add_order_to_list` and just putting it in `add`.

[Here](https://softwareengineering.stackexchange.com/a/386674) is a stackoverflow answer that elaborates on this a bit.

Are there other places you are doing this in the code?